docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,133 @@
-| `FocusLost`         | Emitted when the control loses key input focus.           |
-| `HoverEvent`        | Emitted when the hover input is received.                 |
-| `KeyEvent`          | Emitted when the key event is received.                   |
-| `LayoutDirectionChanged` | Emitted when  the layout direction property of this or a parent view is changed. |
-| `Relayout`          | Emitted after the size has been set on the view during relayout. |
-| `RemovedFromWindow` | Emitted after the view has been disconnected from the window. |
-| `ResourcesLoaded`   | Emitted after all resources required by a view are loaded and ready. |
-| `TouchEvent`        | Emitted when the touch input is received.                 |
-| `VisibilityChanged` | Emitted when the visible property of this or a parent view is changed. |
-| `WheelEvent`        | Emitted when the wheel event is received.                 |
+In addition, if the property `ParentOrigin` is changed to **BottomRight**, then the position of `View` is moved to (5, 3) from the **BottomRight** position of its parent. NUI provides predefined `ParentOrigin` as shown in the following figure:
 
-<a name="2"></a>
-## Setting the Background Color
+**Figure: The position of predefined ParentOrigin**
 
-You can set a background color for a UI component. To set a red background for a component:
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/view_Figure_ParentOrigin_PivotPoint.png">
+</td>
+</tr>
+</table>
 
-```
-View view = new View();
-view.Size2D = new Size2D(200, 200);
-view.BackgroundColor = Color.Red;
-```
+The following figure shows three cases of parent `View` and its child `View`. Parent `View` is blue and child `View` is red. In the following examples, all of the red `View` is identically located at (3, 5) of its parent coordinate system that is based on `ParentOrigin` property of the child `View`.
 
-**Figure: View with a red background**
+**Figure: The final position with different ParentOrigin**
 
-![View with a red background](./media/background_control_color.png)
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/view_Figure_Position.png" width="100%">
+</td>
+</tr>
+</table>
 
-You can handle all existing views similarly. For example, to set the background color for a TextLabel:
+Alternatively, you can use an arbitrary `Position` type value defined in the unit coordinates to set `ParentOrigin` without predefined values.
+
+`Orientation` of `View` is the rotation from its default orientation. And `Scale` of `View` is the size ratio between the size to be rendered and the default size. For these two transforms, `View` is rotated and scaled around [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint). NUI also provides predefined presets of `PivotPoint` identical to `ParentOrigin`. Alternatively, you can use custom values.
```

Comment:
It is not recommended to start a sentence with "And".
Please remove "And".

`Orientation` of `View` is the rotation from its default orientation. `Scale` of `View` is the size ratio between the size to be rendered and the default size

Reply:
I fixed it. Thank you.